client/stylesheets/_toCat.scss

```diff
@@ -4511,3 +4511,140 @@ On click the images will be displayed at normal size to complete the effect
 .profileSettingsButton input:hover {
     background-color: #f73c37 !important;
 }
+
+/* View Feedback */
+
+.profileUserFeedbackView {
+    margin: 0 !important;
+    margin: auto !important;
+    width: 60%;
+    max-width: 1100px;
+}
+
+.profileUserFeedbackViewSymbols {
```

Comment:
Selector `profileUserFeedbackViewSymbols` should be written in lowercase with hyphens